.devcontainer/ubuntu/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+  name: "netty-on-ubuntu",
+  containerEnv: {
+    CPATH: "/usr/lib/jvm/java-21-openjdk-arm64/include/:/usr/lib/jvm/java-21-openjdk-arm64/include/linux/"
+  },
+  build: {
+    dockerfile: "Dockerfile",
+  },
+  "securityOpt": [ "seccomp=unconfined" ]
```

Comment:
any need to tweak the memory to allocate the ring buffer itself?

Reply:
nope